CreateSurfaceWindowsImage.ps1

```diff
@@ -969,8 +1022,40 @@ Function Get-LatestUpdates
             }
         }
         $Date = Get-Date -Format "yyyy-MM"
-        $LoopBreak = $null
-        
+        $LoopBreak = $null   
+    }
+    If ($OutOfBand)

```

Comment:
This var is never set by anything so it'll always be $false

I think we want $OOB, though I prefer $OutOfBand